### ISSUE TITLE
flexmock raises "undefined method 'teardown'" on minitest 

### ISSUE THREAD

**Issue by zhon** [jimweirich/flexmock#14]
``` ruby
require "minitest/autorun"
require 'flexmock/test_unit'

class SimpleTest < MiniTest::Unit::TestCase
  def test_flexmock
    flexmock()
  end
end
```

results in the following:

```
undefined method `teardown' for class `Test::Unit::TestCase' (NameError)
from D:/tools/Ruby193/lib/ruby/gems/1.9.1/gems/flexmock-1.3.1/lib/flexmock/test_unit.rb:16:in `<module:Unit>'
```

This is the case for ruby1.9.3 and ruby2.0.0 with minitest 4.1.0

**Comment by zhon** [comment]
Of course the include method still works:

``` Ruby
require "minitest/autorun"
require 'flexmock'

class SimpleTest < MiniTest::Unit::TestCase
  include FlexMock
# ...
```

**Comment by jimweirich** [comment]
There is no teardown in the Test::Unit::TestCase when using MiniTest.  Fixed in 1.3.2.
